oaknational/Oak-Web-Application
# Feat/pupil 844/sign post take down

## Description

Music year: {owa_music_year}

- List of changes
finally! display banner on lesson listing and lesson pages for units with lessons which will be expired in august

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2632--oak-web-application.netlify.thenational.academy/pupils/programmes/english-primary-year-3-l/units/the-day-the-crayons-quit-persuasive-writing-0c4a/lessons
2. you should see banner with lesson removal warnings
3. Click on "To share and discuss a story"
4. You should see banner saying the lesson will be removed and link to support page

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
